flexx/app/_tornadoserver.py

```diff
@@ -676,19 +676,17 @@ def check_origin(self, origin):
         #WebSocketHandler.check_origin
         
         serving_host = self.request.headers.get("Host")
-        serving_hostname = serving_host.split(':')[0]
-        connecting_host = urlparse(origin).netloc
-        connecting_hostname = connecting_host.split(':')[0]
+        serving_hostname = serving_host.split(':')[0].lower()
+        connecting_host = urlparse(origin).netloc.lower()
+
```

Comment:
I'm not sure about .lower() the host names, but I think it could be useful in some cases.